pkg/html/pages/style.css

```diff
@@ -180,14 +180,14 @@ table td, table th {
 }
 
 .list_table .stat_name {
-	width: 150pt;
-	max-width: 150pt;
+	width: 180pt;
+	max-width: 180pt;
```

Comment:
We seem to do this for most styles. The reason is to avoid sudden changes in width and keep them consistent across different tables.
Say, if we render simple and expert metrics in separate tables, one below the other, then they may have different widths.
If if you load 2 pages at different time for comparison, they may have different widths.

Reply:
> Say, if we render simple and expert metrics in separate tables, one below the other

Why would these be in separate tables?

Reply:
Just a hypothetical UI option. Say, instead of the current expert/non-export mode with page reloading, we have a table with export stats that can be collapsed.
But I am not very strong about it, can remove if you feel we should.